contractcourt/htlc_success_resolver.go

```diff
@@ -2,7 +2,6 @@ package contractcourt
 
 import (
 	"encoding/binary"
-	"fmt"
```

Comment:
should this be a fixup (or two) instead?

Reply:
Yes, it should! Fixed(up) and squashed!